### PR TITLE
New test coverage tools

### PR DESCRIPTION
**Pull request by EugeneElkin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 2.1.0
+* New test coverage tools
+
+## 2.0.0
```
Comment (ignatvilesov) [inline CHANGELOG.md]: Keep it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## 2.1.0
```
Comment (ignatvilesov) [inline CHANGELOG.md]: Please do not increase visual version

File: package.json [hunk 3]
```diff
@@ -37,12 +37,14 @@
...
     "css-loader": "1.0.0",
     "d3": "5.7.0",
+    "istanbul-instrumenter-loader": "^3.0.1",
     "jasmine": "2.6.0",
     "jasmine-jquery": "2.1.1",
     "jquery": "3.2.1",
     "karma": "2.0.4",
     "karma-chrome-launcher": "2.2.0",
     "karma-coverage": "1.1.2",
+    "karma-coverage-istanbul-reporter": "^2.0.4",
```
Comment (ignatvilesov) [inline package.json]: Are old package still in use? Otherwise, let remove useless packages from dependencies

File: CHANGELOG.md (hunk 1 above)
Comment (EugeneElkin) [reply CHANGELOG.md]: Done!

File: package.json (hunk 3 above)
Comment (EugeneElkin) [reply package.json]: Done!

**Pull request merged**